tests/unit/modules/network/sonic/fixtures/sonic_vrfs.yaml

```diff
@@ -79,17 +79,10 @@ deleted_01:
                     - id: Eth1/1
                     - id: Eth1/2
               - name: mgmt
-                interfaces:
```

Comment:
Deletion of Ethernet interfaces from the Management VRF is a permitted operation.

Reply:
Thanks Done!

Reply:
Regression tested, the result and log are reloaded!